src/hb-aat-layout-lcar-table.hh

```diff
@@ -55,8 +55,7 @@ struct lcar
     if (caret_count)
     {
       hb_array_t<const HBINT16> arr = array.sub_array (start_offset, caret_count);
-      unsigned int count = arr.length;
-      for (unsigned int i = 0; i < count; ++i)
+      for (unsigned int i = 0; i < arr.length; ++i)
 	switch (format)
```

Comment:
This switch is so badly places.  Should have been subformats.  Fix later please.